model.js

```diff
@@ -4,10 +4,15 @@ const config = require('config');
 const ttl = (config.craigslist && config.craigslist.ttl) || 60 * 60;
 const request = require('request').defaults({ gzip: true });
 const types = require('./mappings/types.js');
+const idField = 'featureId';
 
 module.exports = function() {
+  // Adding "idField" as instance property gives koop-core ability to determine presence of idField during provider 
+  // registration. 
+  this.idField = idField
```

Comment:
I don't really think you should set this in 3 places, once in metadata would be enough